src/GraphQL/FieldHelper/DataObjectFieldHelper.php

```diff
@@ -168,12 +168,12 @@ public function getFieldDefinitionFromKey($class, $key, &$container = null)
             $brickDefinition = Definition::getByKey($brickType);
 
             if ($brickDescriptor) {
-                $fieldDefinition = $brickDefinition->getFieldDefinition($brickKey);
```

Comment:
It seems the getFieldDefinition() method of the object brick definition also get localized field definitions. So the $container variable isn't set for localized fields.

Reply:
You are right, that´s why we unified the behavior for pimcore 11 (see your other comment). Thanks!